snowplow-android-core/build.gradle

```diff
@@ -0,0 +1,48 @@
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 21
+    buildToolsVersion "21.1.2"
+
+    defaultConfig {
+        minSdkVersion 11
+        targetSdkVersion 21
+        versionCode 1
+        versionName "1.0"
+    }
+    buildTypes {
+        debug {
+            buildConfigField "String", "TRACKER_LABEL", "\"andr-$project.version\""
+            buildConfigField "String", "TRACKER_VERSION", "\"$project.version\""
+        }
+        release {
```

Comment:
@jbeemster I found that I could do this instead of generating a Version.java and thought it was a nice way to take that out. However, there may be another reason for Version.java that I'm not aware of and we should keep it. Thoughts?